### PR TITLE
Adds Kubernetes App update info after succesfull migration

### PR DESCRIPTION
**Pull request by perk-sumo**
###### Description

Fill in your description here.

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/docs/v1_migration_doc.md [hunk 1]
```diff
@@ -201,3 +202,6 @@ sed 's/cluster kubernetes/cluster <CLUSTER_NAME>/g'  >> fluentd-sumologic.yaml
 - Follow steps mentioned [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/docs/Non_Helm_Installation.md#deploy-prometheus) to deploy Prometheus.
 ##### 2.3: Deploy Fluent Bit
 - Follow steps mentioned [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/docs/Non_Helm_Installation.md#deploy-fluentbit) to deploy Fluent Bit.
+
+## Kubernetes App dashboard update
+After successful migration please make sure to update your Kubernetes App dashboard to the latest version.
```
Comment (frankreno) [inline deploy/docs/v1_migration_doc.md:207]: Let's link to the help docs as well.

```suggestion
After successful migration please make sure to [reinstall your Kubernetes App](https://help.sumologic.com/07Sumo-Logic-Apps/10Containers_and_Orchestration/Kubernetes/Install_the_Kubernetes_App_and_view_the_Dashboards) to the latest version.
```

**Comment by frankreno** [comment]
One other thing, we also should have a release note that explains that we did remove unnecessary metrics from a collection that were not used by the app and link to the PR.

**Review by sumo-drosiek — COMMENTED** [review]
LGTM

File: deploy/docs/v1_migration_doc.md (hunk 1 above)
Comment (vsinghal13) [reply deploy/docs/v1_migration_doc.md:207]: +1

**Review by sumo-drosiek — APPROVED** [review]

**Review by pmalek-sumo — APPROVED** [review]

**Pull request merged**